REST_API.md

```diff
@@ -183,9 +186,15 @@ The following synonyms and shortened values for standard operations are supporte
 - `delete` - `d`
 - `manage` (any operation is allowed) - `m`, `admin`
 
+`include-patterns` is an optional argument (effectively is `["**"]` by default) 
+which allows specifying path patterns to which permissions are applied.
+Path patterns are specified by Ant-like expressions.
+
 Possible responses:
 - `200 OK` when permissions were added successfully
 - `500 INTERNAL ERROR` in the case of unexpected server error
+- `400 BAD REQUEST` in the cases when repository name does not exist 
+ or `include-patterns` format is invalid.
```

Comment:
@olegmoz let's remove point at the end of the sentence, they are not required in the lists